infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/environment/OpenShiftEnvironment.java

```diff
@@ -42,6 +44,21 @@ public OpenShiftEnvironment(KubernetesEnvironment k8sEnv) {
     this.routes = new HashMap<>();
     setType(TYPE);
     setAttributes(k8sEnv.getAttributes());
+    setCommands(k8sEnv.getCommands());
+  }
+
+  public OpenShiftEnvironment(
+      InternalEnvironment internalEnvironment,
```

Comment:
You might also consider adding constructor that passes k8s environment instead of internal one

Reply:
As far as I see, it is already there https://github.com/eclipse/che/blob/d2382a04dada23236237c30b07a80f4dadd522a9/infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/environment/OpenShiftEnvironment.java#L42-L48